ckanext/harvest/templates/snippets/harvest_source_job_details.html

```diff
@@ -0,0 +1,70 @@
+
+{% if source.get(state, '').startswith('draft') %}
+    <span class="label label-info">{{ _('Draft') }}</span>
+{% elif source.get(state, '').startswith('deleted') %}
+    <span class="label label-important">{{ _('Deleted') }}</span>
+{% endif %}
+
+{% set stats = job.stats %}
+
+{% if job.status == 'Finished' %}
+    <p>
+    <span class="label label-important" data-diff="error">
+      {% if 'errored' in stats and stats['errored'] > 0 %}
+          {{ h.link_to("{} {}".format(stats['errored'], _('errors')), h.url_for('harvest_job_show', source=source.get('id'), id=job['id'])) }}
+      {% else %}
+          0 {{ _('errors') }}
+      {% endif %}
+    </span>
+        {% for action in ['added', 'updated', 'deleted', 'not modified'] %}
+            <span class="label" data-diff="{{ action }}">
+        {% if action in stats and stats[action] > 0 %}
+            {{ stats[action] }}
+        {% else %}
+            0
+        {% endif %}
+                {{ _(action) }}
+      </span>
+        {% endfor %}
+    </p>
+{% endif %}
+
+<h3 class="hide-heading">{{ _('Details') }}</h3>
+<table class="table table-striped table-bordered table-condensed">
+    <colgroup>
+        <col width="15">
+        <col width="85">
+    </colgroup>
+    <tr>
+        <th>{{ _('Created') }}</th>
+        <td>
+        <span class="automatic-local-datetime" data-datetime="{{ h.render_datetime(job.created, date_format='%Y-%m-%dT%H:%M:%S%z') }}">
+            {{ h.render_datetime(job.created, with_hours=True) }}
+        </span>
+        </td>
+    </tr>
+    <tr>
+        <th>{{ _('Started') }}</th>
+        <td>
+        <span class="automatic-local-datetime" data-datetime="{{ h.render_datetime(job.gather_started, date_format='%Y-%m-%dT%H:%M:%S%z') }}">
+            {{ h.render_datetime(job.gather_started, with_hours=True) }}
+        </span>
+        </td>
+    </tr>
+    <tr>
+        <th>{{ _('Finished') }}</th>
+        <td>
+        <span class="automatic-local-datetime" data-datetime="{{ h.render_datetime(job.finished, date_format='%Y-%m-%dT%H:%M:%S%z') }}">
+            {{ h.render_datetime(job.finished, with_hours=True) }}
+        </span>
+        </td>
+    </tr>
```

Comment:
The rows for job status should be hidden if there are no jobs or if a public user is looking at it.

Reply:
done